From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jihoon Han <rapid_renard@renard.kr>
Date: Mon, 11 Dec 2023 04:47:08 +0900
Subject: [PATCH] HACK: configs: meson64: prevent stdout/stderr on
 videoconsole

Several devices have CONFIG_DM_VIDEO enabled which causes stdout/stderr
to appear on videoconsole, so remove videoconsole from STDOUT so that
early u-boot boot remains silent unless using the uart/serial console.

Signed-off-by: Jihoon Han <rapid_renard@renard.kr>
---
 include/configs/meson64.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/meson64.h b/include/configs/meson64.h
index d21899c335..2ac866c9ca 100644
--- a/include/configs/meson64.h
+++ b/include/configs/meson64.h
@@ -23,11 +23,11 @@
 		230400, 250000, 460800, 500000, 1000000, 2000000, 4000000, \
 		8000000 }
 
 /* For splashscreen */
 #ifdef CONFIG_VIDEO
-#define STDOUT_CFG "vidconsole,serial"
+#define STDOUT_CFG "serial"
 #else
 #define STDOUT_CFG "serial"
 #endif
 
 #ifdef CONFIG_USB_KEYBOARD
-- 
Created with Armbian build tools https://github.com/armbian/build

